--- newt-0.52.16/Makefile.in.orig	2013-09-01 20:22:06.877393120 +0200
+++ newt-0.52.16/Makefile.in	2013-09-01 20:29:26.177374684 +0200
@@ -69,19 +69,19 @@
 all:	$(TARGET) $(SNACKSO)
 
 test:	test.o $(LIBNEWT)
-	$(CC) -g -o test test.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
+	$(CC) $(CFLAGS) -o test test.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
 
 testgrid:	testgrid.o $(LIBNEWT)
-	$(CC) -g -o testgrid testgrid.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
+	$(CC) $(CFLAGS) -o testgrid testgrid.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
 
 testtree:	testtree.o $(LIBNEWT)
-	$(CC) -g -o testtree testtree.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
+	$(CC) $(CFLAGS) -o testtree testtree.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
 
 showchars:	showchars.o $(LIBNEWT)
-	$(CC) -g -o showchars showchars.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
+	$(CC) $(CFLAGS) -o showchars showchars.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
 
 showkey:	showkey.o $(LIBNEWT)
-	$(CC) -g -o showkey showkey.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
+	$(CC) $(CFLAGS) -o showkey showkey.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
 
 _snackmodule.$(SOEXT):   snackmodule.c $(LIBNEWTSH)
 	@[ -n "$(PYTHONVERS)" ] && for ver in $(PYTHONVERS); do \
@@ -104,10 +104,10 @@
 	touch $@
 
 whiptail: $(NDIALOGOBJS) $(LIBNEWTSH)
-	$(CC) -g -o whiptail $(NDIALOGOBJS) -L. $(LDFLAGS) -lnewt $(LIBS) -lpopt
+	$(CC) $(CFLAGS) -o whiptail $(NDIALOGOBJS) -L. $(LDFLAGS) -lnewt $(LIBS) -lpopt
 
 whiptcl.$(SOEXT): $(WHIPTCLOBJS) $(LIBNEWTSH)
-	$(CC) -shared $(SHCFLAGS) $(LDFLAGS) -o whiptcl.$(SOEXT) $(WHIPTCLOBJS) -L. -lnewt  $(LIBTCL) -lpopt $(LIBS)
+	$(CC) -shared $(SHCFLAGS) $(CFLAGS) $(LDFLAGS) -o whiptcl.$(SOEXT) $(WHIPTCLOBJS) -L. -lnewt  $(LIBTCL) -lpopt $(LIBS)
 
 $(LIBNEWT): $(LIBOBJS)
 	ar rv $@ $^
@@ -122,7 +122,7 @@
 		$(SHAREDDIR)/*.o *.$(SOEXT)*
 
 depend:
-	$(CPP) $(CFLAGS) $(CPPFLAGS) -M $(SOURCES) > .depend
+	$(CPP) $(CPPFLAGS) -M $(SOURCES) > .depend
 
 $(SHAREDDIR):
 	mkdir -p $(SHAREDDIR)
@@ -130,7 +130,7 @@
 sharedlib: $(LIBNEWTSH)
 
 $(LIBNEWTSH): $(SHAREDDIR) $(SHAREDOBJS)
-	$(CC) -shared -o $(LIBNEWTSH) $(SHLIBFLAGS) $(SHAREDOBJS) $(LDFLAGS) $(LIBS)
+	$(CC) -shared -o $(LIBNEWTSH) $(CFLAGS) $(SHLIBFLAGS) $(SHAREDOBJS) $(LDFLAGS) $(LIBS)
 	ln -fs $(LIBNEWTSONAME) libnewt.$(SOEXT)
 	ln -fs $(LIBNEWTSH) $(LIBNEWTSONAME)
 
