--- newt-0.52.12/Makefile.in.orig	2010-11-05 17:25:34.729541330 +0200
+++ newt-0.52.12/Makefile.in	2010-11-05 17:29:33.376208002 +0200
@@ -54,19 +54,19 @@
 all:	$(TARGET) $(SNACKSO)
 
 test:	test.o $(LIBNEWT)
-	$(CC) -g -o test test.o $(LIBNEWT) $(LIBS)
+	$(CC) $(CFLAGS) $(CPPFLAGS) -o test test.o $(LIBNEWT) $(LIBS)
 
 testgrid:	testgrid.o $(LIBNEWT)
-	$(CC) -g -o testgrid testgrid.o $(LIBNEWT) $(LIBS)
+	$(CC) $(CFLAGS) $(CPPFLAGS) -o testgrid testgrid.o $(LIBNEWT) $(LIBS)
 
 testtree:	testtree.o $(LIBNEWT)
-	$(CC) -g -o testtree testtree.o $(LIBNEWT) $(LIBS)
+	$(CC) $(CFLAGS) $(CPPFLAGS) -o testtree testtree.o $(LIBNEWT) $(LIBS)
 
 showchars:	showchars.o $(LIBNEWT)
-	$(CC) -g -o showchars showchars.o $(LIBNEWT) $(LIBS)
+	$(CC) $(CFLAGS) $(CPPFLAGS) -o showchars showchars.o $(LIBNEWT) $(LIBS)
 
 showkey:	showkey.o $(LIBNEWT)
-	$(CC) -g -o showkey showkey.o $(LIBNEWT) $(LIBS)
+	$(CC) $(CFLAGS) $(CPPFLAGS) -o showkey showkey.o $(LIBNEWT) $(LIBS)
 
 _snackmodule.so:   snackmodule.c $(LIBNEWTSH)
 	for ver in $(PYTHONVERS) ; do \
@@ -77,16 +77,16 @@
 	touch $@
 
 whiptail: $(NDIALOGOBJS) $(LIBNEWTSH)
-	$(CC) -g -o whiptail $(NDIALOGOBJS) -L . -lnewt $(LIBS) -lpopt
+	$(CC) $(CFLAGS) $(CPPFLAGS) -o whiptail $(NDIALOGOBJS) -L . -lnewt $(LIBS) -lpopt
 
 whiptcl.o:   whiptcl.c
-	gcc -fPIC $(CFLAGS) -c whiptcl.c
+	$(CC) -fPIC $(CFLAGS) $(CPPFLAGS) -c whiptcl.c
 
 dialogboxes.o:   dialogboxes.c
-	gcc -fPIC $(CFLAGS) -c dialogboxes.c
+	$(CC) -fPIC $(CFLAGS) -c dialogboxes.c
 
 whiptcl.so: $(WHIPTCLOBJS) $(LIBNEWTSH)
-	$(CC) -shared $(SHCFLAGS) -o whiptcl.so $(WHIPTCLOBJS) -L . -lnewt  $(LIBTCL) -lpopt
+	$(CC) -shared $(SHCFLAGS) $(CFLAGS) $(CPPFLAGS) -o whiptcl.so $(WHIPTCLOBJS) -L . -lnewt  $(LIBTCL) -lpopt
 
 $(LIBNEWT): $(LIBOBJS)
 	ar rv $@ $^
@@ -101,14 +101,15 @@
 		$(SHAREDDIR)/*.o *.so* *.pc
 
 depend:
-	$(CPP) $(CFLAGS) $(CPPFLAGS) -M $(SOURCES) > .depend
+	$(CPP) $(CPPFLAGS) -M $(SOURCES) > .depend
 
-$(SHAREDDIR):
+.shared.dir:
 	mkdir -p $(SHAREDDIR)
+	touch .shared.dir
 
 sharedlib: $(LIBNEWTSH)
 
-$(LIBNEWTSH): $(SHAREDDIR) $(SHAREDOBJS)
+$(LIBNEWTSH): .shared.dir $(SHAREDOBJS)
 	$(CC) -shared -o $(LIBNEWTSH) $(SHLIBFLAGS) -Wl,-soname,$(LIBNEWTSONAME) $(SHAREDOBJS) $(LIBS)
 	ln -fs $(LIBNEWTSONAME) libnewt.so
 	ln -fs $(LIBNEWTSH) $(LIBNEWTSONAME)
