--- windows.c.orig	2003-06-30 20:23:27.000000000 +0000
+++ windows.c	2003-06-30 20:24:50.000000000 +0000
@@ -167,7 +167,7 @@
 	buttonName = va_arg(args, char *);
     }
 
-    va_end(button1);
+    va_end(args);
 
     buttonBar = newtCreateGrid(numButtons, 1);
     for (i = 0; i < numButtons; i++) {
@@ -224,7 +224,7 @@
 	buttonName = va_arg(args, char *);
     }
 
-    va_end(button1);
+    va_end(args);
 
     buttonBar = newtCreateGrid(numButtons, 1);
     for (i = 0; i < numButtons; i++) {
