--- newt-0.50.34.orig/Makefile.in	2004-12-21 08:45:53.000000000 +0100
+++ newt-0.50.34/Makefile.in	2004-12-21 08:55:15.992429456 +0100
@@ -67,6 +67,12 @@
 whiptail: $(NDIALOGOBJS) $(LIBNEWTSH)
 	gcc -g -o whiptail $(NDIALOGOBJS) -L . $(LIBNEWTSH) $(LIBS) -lpopt
 
+whiptcl.o:   whiptcl.c
+	gcc -fPIC $(CFLAGS) -c whiptcl.c
+
+dialogboxes.o:   dialogboxes.c
+	gcc -fPIC $(CFLAGS) -c dialogboxes.c
+
 whiptcl.so: $(WHIPTCLOBJS) $(LIBNEWTSH)
 	gcc -shared $(SHCFLAGS) -o whiptcl.so $(WHIPTCLOBJS) -L . $(LIBNEWTSH) -ltcl -lslang -lpopt -lm
 
