--- newt-0.50.34/Makefile.in.orig	Sat Sep 15 00:02:04 2001
+++ newt-0.50.34/Makefile.in	Fri Jan 18 09:33:55 2002
@@ -30,8 +30,9 @@
 libdir = $(prefix)/lib
 bindir = $(prefix)/bin
 ARCHNAME = $(shell uname -m | sed 's/i.86/i386/')
-pythondir = $(prefix)/lib/python2.2
-pythonbindir = $(prefix)/lib/python2.2/lib-dynload
+pythondir = $(shell python -c "import distutils.sysconfig; print distutils.sysconfig.get_config_var('DESTLIB');")
+pythonbindir = $(shell python -c "import distutils.sysconfig; print distutils.sysconfig.get_config_var('DESTSHARED');")
+pythonincludedir = $(shell python -c 'from distutils.sysconfig import get_python_inc; print get_python_inc();')
 
 #--------------------------------------
 
@@ -61,7 +62,7 @@
 	gcc --shared $(SHCFLAGS) -o _snackmodule.so snackmodule.o -L . $(LIBNEWTSH)
 
 snackmodule.o:   snackmodule.c
-	gcc -I/usr/include/python2.2 -fPIC $(CFLAGS) -c snackmodule.c
+	gcc -I$(pythonincludedir) -fPIC $(CFLAGS) -c snackmodule.c
 
 whiptail: $(NDIALOGOBJS) $(LIBNEWTSH)
 	gcc -g -o whiptail $(NDIALOGOBJS) -L . $(LIBNEWTSH) $(LIBS) -lpopt
