--- newt-0.51.6/Makefile.in.orig	2005-09-22 21:13:48.098787000 +0200
+++ newt-0.51.6/Makefile.in	2005-09-22 21:17:41.344329128 +0200
@@ -16,6 +16,7 @@
 
 WHIPTCLSO=
 #WHIPTCLSO=whiptcl.so
+SNACKSO=_snackmodule.so
 
 PROGS = test whiptail $(WHIPTCLSO) testgrid testtree showchars showkey
 TESTOBJS = test.o
@@ -49,7 +50,7 @@
 TARGET=depend $(PROGS)
 endif
 
-all:	$(TARGET) _snackmodule.so
+all:	$(TARGET) $(SNACKSO)
 
 test:	$(TESTOBJS) $(LIBNEWT)
 	athlon-pld-linux-gcc -g -o test $(TESTOBJS) $(LIBNEWT) $(LIBS) -static
@@ -126,13 +127,13 @@
 	install -m 644 $(LIBNEWT) $(instroot)/$(libdir)
 	install -m 755 whiptail $(instroot)/$(bindir)
 
-install-sh: sharedlib $(WHIPTCLSO) _snackmodule.so
+install-sh: sharedlib $(WHIPTCLSO) $(SNACKSO)
 	[ -d $(instroot)/$(libdir) ] || install -m 755 -d $(instroot)/$(libdir)
 	install -m 755 $(LIBNEWTSH) $(instroot)/$(libdir)
 	ln -sf $(LIBNEWTSH) $(instroot)/$(libdir)/$(LIBNEWTSONAME)
 	ln -sf $(LIBNEWTSONAME) $(instroot)/$(libdir)/libnewt.so
 	[ -n "$(WHIPTCLSO)" ] && install -m 755 whiptcl.so $(instroot)/$(libdir) || :
-	for ver in $(PYTHONVERS) ; do \
+	[ -z "$(SNACKSO)" ] || for ver in $(PYTHONVERS) ; do \
 	   [ -d $(instroot)/$(libdir)/$$ver/site-packages ] || install -m 755 -d $(instroot)/$(libdir)/$$ver/site-packages ;\
 	   install -m 755 $$ver/_snackmodule.so $(instroot)/$(libdir)/$$ver/site-packages ;\
 	   install -m 755 snack.py $(instroot)/$(libdir)/$$ver/site-packages ;\
