--- newt-0.52.16/Makefile.in.orig	2013-09-01 20:20:06.584064835 +0200
+++ newt-0.52.16/Makefile.in	2013-09-01 20:21:46.550727306 +0200
@@ -20,6 +20,7 @@
 else
 WHIPTCLSO =
 endif
+SNACKSO = _snackmodule.$(SOEXT)
 
 PROGS = test whiptail $(WHIPTCLSO) testgrid testtree showchars showkey
 TESTOBJS = test.o testgrid.o testtree.o showchars.o showkey.o
@@ -65,7 +66,7 @@
 TARGET=depend $(PROGS)
 endif
 
-all:	$(TARGET) _snackmodule.$(SOEXT)
+all:	$(TARGET) $(SNACKSO)
 
 test:	test.o $(LIBNEWT)
 	$(CC) -g -o test test.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
@@ -149,7 +150,7 @@
 	make -C po datadir=$(instroot)/$(datadir) install
 	install -m 644 libnewt.pc $(instroot)/$(pkgconfigdir)
 
-install-sh: sharedlib $(WHIPTCLSO) _snackmodule.$(SOEXT)
+install-sh: sharedlib $(WHIPTCLSO) $(SNACKSO)
 	[ -d $(instroot)/$(libdir) ] || install -m 755 -d $(instroot)/$(libdir)
 	install -m 755 $(LIBNEWTSH) $(instroot)/$(libdir)
 	ln -sf $(LIBNEWTSONAME) $(instroot)/$(libdir)/libnewt.$(SOEXT)
