diff -urN newt-0.51.6.org/Makefile.in newt-0.51.6/Makefile.in
--- newt-0.51.6.org/Makefile.in	2003-11-06 07:56:47.000000000 +0100
+++ newt-0.51.6/Makefile.in	2005-09-16 10:08:00.388120152 +0200
@@ -123,7 +123,8 @@
 install-sh: sharedlib $(WHIPTCLSO) _snackmodule.so
 	[ -d $(instroot)/$(libdir) ] || install -m 755 -d $(instroot)/$(libdir)
 	install -m 755 $(LIBNEWTSH) $(instroot)/$(libdir)
-	ln -sf $(LIBNEWTSH) $(instroot)/$(libdir)/libnewt.so
+	ln -sf $(LIBNEWTSH) $(instroot)/$(libdir)/$(LIBNEWTSONAME)
+	ln -sf $(LIBNEWTSONAME) $(instroot)/$(libdir)/libnewt.so
 	[ -n "$(WHIPTCLSO)" ] && install -m 755 whiptcl.so $(instroot)/$(libdir) || :
 	for ver in $(PYTHONVERS) ; do \
 	   [ -d $(instroot)/$(libdir)/$$ver/site-packages ] || install -m 755 -d $(instroot)/$(libdir)/$$ver/site-packages ;\
