--- newt-0.52.17/Makefile.in.orig	2014-02-19 17:56:53.000000000 +0100
+++ newt-0.52.17/Makefile.in	2014-03-01 12:36:24.047966175 +0100
@@ -20,6 +20,7 @@
 else
 WHIPTCLSO =
 endif
+SNACKSO = _snack.$(SOEXT)
 
 PROGS = test whiptail $(WHIPTCLSO) testgrid testtree showchars showkey
 TESTOBJS = test.o testgrid.o testtree.o showchars.o showkey.o
@@ -65,7 +66,7 @@
 TARGET=depend $(PROGS)
 endif
 
-all:	$(TARGET) _snack.$(SOEXT)
+all:	$(TARGET) $(SNACKSO)
 
 test:	test.o $(LIBNEWT)
 	$(CC) -g -o test test.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
@@ -149,7 +150,7 @@
 	install -m 644 whiptail.1 $(instroot)/$(man1dir)
 	make -C po datadir=$(instroot)/$(datadir) install
 
-install-sh: sharedlib $(WHIPTCLSO) _snack.$(SOEXT)
+install-sh: sharedlib $(WHIPTCLSO) $(SNACKSO)
 	[ -d $(instroot)/$(libdir) ] || install -m 755 -d $(instroot)/$(libdir)
 	[ -d $(instroot)/$(includedir) ] || install -m 755 -d $(instroot)/$(includedir)
 	[ -d $(instroot)/$(pkgconfigdir) ] || install -m 755 -d $(instroot)/$(pkgconfigdir)
