--- newt-0.52.13/Makefile.in.orig	2011-06-30 20:14:00.860766835 +0200
+++ newt-0.52.13/Makefile.in	2011-06-30 21:14:35.674221877 +0200
@@ -14,6 +14,7 @@
 
 PYTHONVERS = @PYTHONVERS@
 WHIPTCLSO = @WHIPTCLSO@
+SNACKSO = _snackmodule.so
 
 PROGS = test whiptail $(WHIPTCLSO) testgrid testtree showchars showkey
 TESTOBJS = test.o testgrid.o testtree.o showchars.o showkey.o
@@ -59,7 +60,7 @@
 TARGET=depend $(PROGS)
 endif
 
-all:	$(TARGET) _snackmodule.so
+all:	$(TARGET) $(SNACKSO)
 
 test:	test.o $(LIBNEWT)
 	$(CC) -g -o test test.o $(LIBNEWT) $(LDFLAGS) $(LIBS)
@@ -141,7 +142,7 @@
 	make -C po datadir=$(instroot)/$(datadir) install
 	install -m 644 libnewt.pc $(instroot)/$(pkgconfigdir)
 
-install-sh: sharedlib $(WHIPTCLSO) _snackmodule.so
+install-sh: sharedlib $(WHIPTCLSO) $(SNACKSO)
 	[ -d $(instroot)/$(libdir) ] || install -m 755 -d $(instroot)/$(libdir)
 	install -m 755 $(LIBNEWTSH) $(instroot)/$(libdir)
 	ln -sf $(LIBNEWTSONAME) $(instroot)/$(libdir)/libnewt.so
