diff -durN -x '*~' newt-0.50.34.orig/Makefile.in newt-0.50.34/Makefile.in
--- newt-0.50.34.orig/Makefile.in	2004-12-19 15:14:00.000000000 +0100
+++ newt-0.50.34/Makefile.in	2004-12-19 15:16:31.972344880 +0100
@@ -30,9 +30,6 @@
 libdir = $(prefix)/lib
 bindir = $(prefix)/bin
 ARCHNAME = $(shell uname -m | sed 's/i.86/i386/')
-pythondir = $(shell python -c "import distutils.sysconfig; print distutils.sysconfig.get_config_var('DESTLIB');")
-pythonbindir = $(shell python -c "import distutils.sysconfig; print distutils.sysconfig.get_config_var('DESTSHARED');")
-pythonincludedir = $(shell python -c 'from distutils.sysconfig import get_python_inc; print get_python_inc();')
 
 #--------------------------------------
 
@@ -47,7 +44,7 @@
 TARGET=depend $(PROGS)
 endif
 
-all:	$(TARGET) _snackmodule.so
+all:	$(TARGET)
 
 test:	$(TESTOBJS) $(LIBNEWT)
 	gcc -g -o test $(TESTOBJS) $(LIBNEWT) $(LIBS) -static
@@ -58,12 +55,6 @@
 testtree:	testtree.o $(LIBNEWT)
 	gcc -g -o testtree testtree.o $(LIBNEWT) $(LIBS)
 
-_snackmodule.so:   snackmodule.o $(LIBNEWTSH)
-	gcc --shared $(SHCFLAGS) -o _snackmodule.so snackmodule.o -L . $(LIBNEWTSH)
-
-snackmodule.o:   snackmodule.c
-	gcc -I$(pythonincludedir) -fPIC $(CFLAGS) -c snackmodule.c
-
 whiptail: $(NDIALOGOBJS) $(LIBNEWTSH)
 	gcc -g -o whiptail $(NDIALOGOBJS) -L . $(LIBNEWTSH) $(LIBS) -lpopt
 
@@ -108,15 +99,11 @@
 	install -m 644 $(LIBNEWT) $(instroot)/$(libdir)
 	install -s -m 755 whiptail $(instroot)/$(bindir)
 
-install-sh: sharedlib whiptcl.so _snackmodule.so
+install-sh: sharedlib 
 	[ -d $(instroot)/$(libdir) ] || install -m 755 -d $(instroot)/$(libdir)
 	install -s -m 755 $(LIBNEWTSH) $(instroot)/$(libdir)
 	ln -sf $(LIBNEWTSH) $(instroot)/$(libdir)/$(LIBNEWTSONAME)
 	ln -sf $(LIBNEWTSONAME) $(instroot)/$(libdir)/libnewt.so
-	install -s -m 755 whiptcl.so $(instroot)/$(libdir)
-	[ -d $(instroot)/$(pythonbindir) ] || install -m 755 -d $(instroot)/$(pythonbindir)
-	install -s -m 755 _snackmodule.so $(instroot)/$(pythonbindir)
-	install -m 755 snack.py $(instroot)/$(pythondir)
 
 configure:  configure.in
 	autoconf
