diff -Nur newt-0.50.5.ORIG/textbox.c newt-0.50.5/textbox.c
--- newt-0.50.5.ORIG/textbox.c	Sun Oct 22 15:17:17 2000
+++ newt-0.50.5/textbox.c	Sun Oct 22 15:19:39 2000
@@ -332,7 +332,12 @@
    
     for (i = 0; (i + tb->topLine) < tb->numLines && i < c->height; i++) {
 	newtGotorc(c->top + i, c->left);
-	SLsmg_write_string(tb->lines[i + tb->topLine]);
+	SLsmg_write_nstring(tb->lines[i + tb->topLine], c->width);
+    }
+
+    for ( ; i < c->height; i++) {
+	    newtGotorc (c->top + i, c->left);
+	    SLsmg_write_nstring (" ", c->width);
     }
 }
 
